mailpoet/assets/js/src/automation/templates/components/template-list-item.tsx

```diff
@@ -50,7 +53,9 @@ export function TemplateListItem({
           void createWorkflowFromTemplate();
         }}
       >
-        <h2>{template.name} →</h2>
+        {headingTag === 'h3' && <h3>{template.name} →</h3>}
+        {headingTag === 'h2' && <h2>{template.name} →</h2>}
```

Comment:
Should we put a `&nbsp;` before the `→`?

<img width="394" alt="Screenshot 2022-10-27 at 9 31 57" src="https://user-images.githubusercontent.com/141436/198219450-2305fb89-9cee-413f-a3da-1dc3fa57f29e.png">

Reply:
3dae2d4b0

Reply:
After rebase: [95e1290](https://github.com/mailpoet/mailpoet/pull/4461/commits/95e12908bd2a28c2b6ee987567812c544f93d401)